Signal/src/ViewControllers/DebugUI/DebugUIMessages.m

```diff
@@ -1882,6 +1957,200 @@ + (void)testDirectionalFilenamesInThread:(TSThread *)thread
     };
 }
 
++ (void)createFakeQuotedRepliesInThread:(TSThread *)thread
```

Comment:
This debug UI should allow us to work separately on the various aspects of this work.